dask/dataframe/groupby.py

```diff
@@ -1803,6 +1803,11 @@ def cov(self, ddof=1, split_every=None, split_out=1, std=False):
 
         When `std` is True calculate Correlation
         """
+        from dask.dataframe.groupby import SeriesGroupBy
+
+        if isinstance(self, SeriesGroupBy):
+            raise NotImplementedError("cov() not implemented for SeriesGroupBy object")
```

Comment:
Maybe it makes more sense to simply add the following method to `SeriesGroupby`?

```python
def cov(self, *args, **kwargs):
    raise NotImplementedError("cov() not implemented for SeriesGroupBy object")
```

Also, if the logic here is specific to `DataFrameGroupBy`, then maybe it should be moved to `DataFrameGroupBy` entirely (along with `corr`)?